index.js

```diff
@@ -34,17 +33,29 @@ const toBase64 = (extension, data) => {
  * @param swatch
  * @returns {{palette: Array}}
  */
-const toPalette = swatch => {
+const toPalette = swatch => (
```

Comment:
There seem to be duplicate declaration of `toPalette` constant.

Reply:
resolved!